src/http-request.ts

```diff
@@ -0,0 +1,36 @@
+/*!
+ * Copyright 2018 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export interface HttpRequest {
```

Comment:
The name is confusing. This is the type definition for this Stackdriver Logging concept: https://cloud.google.com/logging/docs/reference/v2/rest/v2/LogEntry#HttpRequest

I think I will rename it to `StackdriverHttpRequest` to avoid confusion.